sqlitecluster/SQLiteNode.cpp

```diff
@@ -1482,8 +1496,11 @@ void SQLiteNode::_onMESSAGE(Peer* peer, const SData& message) {
             // processed asynchronously, but that is fine, the final `SUBSCRIBE` message and its response will be
             // processed synchronously.
             SData request = message;
-            request["peerCommitCount"] = (*peer)["CommitCount"];
-            request["peerHash"] = (*peer)["Hash"];
+            uint64_t count = 0;
```

Comment:
nab, explicit definition as 0 seems unnecessary here

Reply:
Yeah, until someone changes something in the future and it breaks cause it's not initialized to anything sane.

Reply:
makes sense, there is another place where you are not init'ing it to 0 explicitly. should we match them?

Reply:
Sure. Comment on that place and I'll add a 0 there.